evaluation/evaluator.go

```diff
@@ -225,17 +228,17 @@ func (e Evaluator) isTargetIncludedOrExcludedInSegment(segmentList []string, tar
 	return false
 }
 
-func (e Evaluator) checkPreRequisite(parent *rest.FeatureConfig, target *rest.Target) (bool, error) {
+func (e Evaluator) checkPreRequisite(fc *rest.FeatureConfig, target *Target) (bool, error) {
 	if e.query == nil {
 		log.Errorf(ErrQueryProviderMissing.Error())
 		return true, ErrQueryProviderMissing
 	}
-	prerequisites := parent.Prerequisites
+	prerequisites := fc.Prerequisites
 	if prerequisites != nil {
 		log.Infof(
 			"Checking pre requisites %v of parent feature %v",
 			prerequisites,
-			parent.Feature)
+			fc.Feature)
```

Comment:
Are we logging the full struct here?   Would it be more helpful to log just the identifier ?